db/seeds.rb

```diff
@@ -5,218 +5,190 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
-def  add_image_to_hike(hike, image_name)
+def add_image_to_hike(hike, image_name)
   image = File.open("db/pictures/#{image_name}")
   extension = image_name.split('.').last
 
   hike.photos.attach(io: image, filename: image_name, content_type: "image/#{extension}")
 end
 
+def create_random_group_hikes(hike)
+  how_many = rand(2..4)
+  random_amount_of_days = 0 
+  how_many.times do |j|
+    random_amount_of_days += rand(hike.length..30)
+    GroupHike.create( 
+      hike: hike,
+      start_at: Date.today + random_amount_of_days,
+      end_at: Date.today + random_amount_of_days + hike.length,
+      chatroom: Chatroom.create(name: hike.name),
+    )
+  end
+end
+
 puts 'cleaning up database'
-GroupHike.destroy_all
+
 Review.destroy_all
 Booking.destroy_all
+GroupHike.destroy_all
+Message.destroy_all
+Chatroom.destroy_all
 Hike.destroy_all
+User.where.not(email: ["anna@hikeasy.fit", "sara@hikeasy.fit", "emma@hikeasy.fit", "vincenzo@hikeasy.fit"]).delete_all
```

Comment:
If you don't want a certain user to be deleted, don't forget to add their emails here.